PyInstaller/depend/bindepend.py

```diff
@@ -622,7 +622,7 @@ def _getImports_macholib(pth):
             #    '../lib\x00\x00')
             cmd_type = command[0].cmd
             if cmd_type == LC_RPATH:
-                rpath = str(command[2])
+                rpath = command[2].decode('ascii')
```

Comment:
Ok, so 'utf-8' seems to be a safe bet for OS X. But not sure about other platforms, particularly Win, which I don't have an easy (pleasant) way to test.

But I'll submit new PR with 'utf-8'.